plugins/javadoc/src/test/kotlin/org/jetbrains/dokka/javadoc/JavadocClasslikeTemplateMapTest.kt

```diff
@@ -312,7 +312,7 @@ internal class JavadocClasslikeTemplateMapTest : AbstractJavadocTemplateMapTest(
             val ownMethods = assertIsInstance<List<*>>(methods["own"]).first()
             val sampleFunction = assertIsInstance<Map<String, Any?>>(ownMethods)
 
-            assertEquals("final &lt;D extends <a href=Generic.html>T</a>&gt; <a href=Generic.html#sampleFunction()>D</a> <a href=Generic.html#sampleFunction()>sampleFunction</a>()", sampleFunction.signatureWithModifiers())
+            assertEquals("final &lt;D extends <a href=Generic.html#Generic>T</a>&gt; <a href=Generic.html#sampleFunction()>D</a> <a href=Generic.html#sampleFunction()>sampleFunction</a>()", sampleFunction.signatureWithModifiers())
```

Comment:
I am not so sure about that. In original javadoc the link is going to `Generic.html` and afaik there is no `Generic` anchor on that page. Why is this change needed?

Reply:
After applying changes, it was value that was calculated. Moreover, Kotlin and Javadoc link providers were producing given link, which seems reasonable, so I dedided to change the test expected value. If we want it to be without anchor, we need to consider why location provider skips the usual link. I will debug it soon so we will know.